packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -424,6 +428,18 @@ export class ProjectService {
         }
     }
 
+    async getJobStatus(jobUuid: string): Promise<Job> {
+        return this.projectModel.getJobstatus(jobUuid);
+    }
+
+    async startJob(jobUuid: string, projectUuid: string): Promise<void> {
+        this.projectModel.updateJobStatus(
```

Comment:
I was really confused because I was looking for where `createJob` happened. But I've just seen that `updateJobStatus` actually inserts a new job?

Reply:
yes, it is using the upsert, so ideally, this is the first time we create the job with the status `STARTED` . Do you prefer me to change the name ?